### ISSUE TITLE
Interface doesn't scale - blurry fonts on Retina - macOS

### ISSUE THREAD

**Issue by kulek1** [microsoft/vscode#75951]
- VSCode Version: 1.35.1
- OS Version: macOS Mojave 10.14.5

Does this issue occur when all extensions are disabled?: Yes

I have a problem with interface scaling on Retina display. 
I'm using external QHD monitor and VSCode is rendered correctly on that display. 
Unfortunately when I disconnect it or just display the editor on macbook pro's display then UI looks blurred and it doesn't scale.

I think this is something related with configuration because when I open VSCode from other location than `Applications` then everything looks fine. 
`Vscode Insiders` version works correctly as well even from `Applications` directory.

That's what I tried:
- `code --disable-gpu`
- `code --disable-gpu --disable-gpu-compositing`
- `defaults write com.microsoft.VSCode.helper CGFontRenderingFontSmoothingDisabled -bool NO` - In my opinion It's a different case than font issue on Mojave
- Downgrade to version ` 1.34`
- Delete VSCode from `Applications` and install the newest version
- Remove `~/.config/Code`
- `rm -rf $HOME/.vscode`
- `rm -rf $HOME/Library/Application\ Support/Code`
- Install clean version without any configs above

Is there another configuration directory that I'm not aware of it?

**Comment by bpasero** [comment]
Can you try to reproduce with our nightly insider builds? You can give our preview releases a try from: https://code.visualstudio.com/insiders/

**Comment by kulek1** [comment]
@bpasero As I mentioned before, I opened `Insider build` from `Applications` directory and it works fine.

Ok, it's quite weird. I've got temporary solution:
When I change the name of the app to anything that's different than `Visual Studio Code` then it is rendered correctly from `Applications` directory. 

I did additional checkup and I renamed insider build to `Visual Studio Code`. It was rendered correctly as well. 

I'm not sure if this is related to macOS rather than VSCode. 🤔
